lib/msal-browser/src/interaction_client/RedirectClient.ts

```diff
@@ -500,6 +511,13 @@ export class RedirectClient extends StandardInteractionClient {
             );
         }
 
+        // Not sure if this is the right place for this logic to live
+        // Read thumbprints from temporary cache if PerformanceClient is not 3P
```

Comment:
`Read thumbprints from temporary cache if PerformanceClient is not 3P` -> `Read thumbprints from temporary cache if PerformanceClient is instance of BrowserPerformanceClient`. Please remove this comment as code below is pretty clear